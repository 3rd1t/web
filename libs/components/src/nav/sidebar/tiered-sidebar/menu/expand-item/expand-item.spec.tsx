import React from "react";
import { render } from "@testing-library/react";

import ExpandIcon from "./expand-item";

describe("ExpandIcon", () => {
  it("should render successfully", () => {
    const { baseElement } = render(<ExpandIcon />);
    expect(baseElement).toMatchSnapshot();
  });
});
