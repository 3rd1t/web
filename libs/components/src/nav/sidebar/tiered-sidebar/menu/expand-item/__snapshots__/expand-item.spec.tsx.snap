// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExpandIcon should render successfully 1`] = `
<body>
  <div>
    <div
      class="w-full"
    >
      <div>
        <header
          class="flex items-center p-2 text-gray-800 rounded-lg hover:bg-gray-100"
        >
          <span
            class="flex items-center justify-center text-lg text-gray-400"
          >
            <svg
              class="w-6 h-6"
              fill="none"
              stroke="currentColor"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M9 5l7 7-7 7"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
              />
            </svg>
          </span>
          <div
            class="flex items-center justify-between space-x-1"
          >
            <span
              class="ml-3"
            >
              Three
            </span>
          </div>
        </header>
        <section
          style="opacity: 1; height: auto;"
        >
          <div
            class="flex flex-wrap"
          >
            <span
              class="p-1 m-1 text-xs bg-gray-100 rounded-sm text-data-700"
            >
              MOM
            </span>
            <span
              class="p-1 m-1 text-xs bg-gray-100 rounded-sm text-data-700"
            >
              MOM
            </span>
            <span
              class="p-1 m-1 text-xs bg-gray-100 rounded-sm text-data-700"
            >
              MOM
            </span>
            <span
              class="p-1 m-1 text-xs bg-gray-100 rounded-sm text-data-700"
            >
              MOM
            </span>
            <span
              class="p-1 m-1 text-xs bg-gray-100 rounded-sm text-data-700"
            >
              MOM
            </span>
            <span
              class="p-1 m-1 text-xs bg-gray-100 rounded-sm text-data-700"
            >
              MOM
            </span>
          </div>
        </section>
      </div>
      <div>
        <header
          class="flex items-center p-2 text-gray-800 rounded-lg hover:bg-gray-100"
        >
          <span
            class="flex items-center justify-center text-lg text-gray-400"
          >
            <svg
              class="w-6 h-6"
              fill="none"
              stroke="currentColor"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M9 5l7 7-7 7"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
              />
            </svg>
          </span>
          <div
            class="flex items-center justify-between space-x-1"
          >
            <span
              class="ml-3"
            >
              Four
            </span>
          </div>
        </header>
      </div>
      <div>
        <header
          class="flex items-center p-2 text-gray-800 rounded-lg hover:bg-gray-100"
        >
          <span
            class="flex items-center justify-center text-lg text-gray-400"
          >
            <svg
              class="w-6 h-6"
              fill="none"
              stroke="currentColor"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M9 5l7 7-7 7"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
              />
            </svg>
          </span>
          <div
            class="flex items-center justify-between space-x-1"
          >
            <span
              class="ml-3"
            >
              Five
            </span>
          </div>
        </header>
      </div>
      <div>
        <header
          class="flex items-center p-2 text-gray-800 rounded-lg hover:bg-gray-100"
        >
          <span
            class="flex items-center justify-center text-lg text-gray-400"
          >
            <svg
              class="w-6 h-6"
              fill="none"
              stroke="currentColor"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M9 5l7 7-7 7"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
              />
            </svg>
          </span>
          <div
            class="flex items-center justify-between space-x-1"
          >
            <span
              class="ml-3"
            >
              Six
            </span>
          </div>
        </header>
      </div>
    </div>
  </div>
</body>
`;
